Content.Shared/_DV/Medical/HormoneComponents.cs

```diff
@@ -23,3 +23,11 @@ public sealed partial class FeminizedComponent : Component, IHormoneComponent {
     [DataField("original")]
     public Sex? Original { get; set; } = null;
 }
+
+[RegisterComponent, NetworkedComponent]
+public sealed partial class AndrogynizedComponent : Component, IHormoneComponent { // imp
+    public Sex Target => Sex.Unsexed;
+
+    [DataField("original")]
```

Comment:
`("original")` is redundant, this exact name is given by the variable already. though i suppose this goes for the rest of the components here..